app/interpreters/schedule_message_interpreter.rb

```diff
@@ -3,7 +3,7 @@
 class ScheduleMessageInterpreter
   attr_reader :schedule, :message
 
-  def initialize(schedule_id, event_uuid)
+  def initialize(schedule_id, event_uuid = nil)
```

Comment:
@hengsokly this class has a dependency coupling, you should consider passing the schedule and event object instead of id/UUID as you will encounter difficulties of writing test spec